recipes/parallel-preprocessor/meta.yaml

```diff
@@ -0,0 +1,80 @@
+# manually update or generated from template
+package:
+    name: parallel-preprocessor
+    version: '0.3'
+source:
+    # calc check sum by command sha256sum
+    sha256: 0f472735203b16e092686003e50e25d2e6dbdeef43da8f14177d6bcb4db86634
+    url: https://github.com/ukaea/parallel-preprocessor/releases/download/v0.3/parallel-preprocessor.zip
+
+build:
+    # number should be generated, or manually update, YYMM pattern here, must be 0 for the first PR
+    number: '0'
+    string: oct_2020
+    skip: true  # [py < 38 or osx]
+
+# libGL needs Core Dependency Tree (CDT) packages
+# https://conda-forge.org/docs/maintainer/knowledge_base.html#special-dependencies
+requirements:
+    run_constrained:
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+    build:
+        - python
+        - cmake
+        - git
+        - ninja
+        - make  # [unix]
+        - {{ compiler("cxx") }}
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('mesa-libegl-devel') }}     # [linux]
+        - {{ cdt('libselinux') }}            # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - {{ cdt('libxcb') }}                # [linux]
+        - {{ cdt('libxext') }}               # [linux]
+        - {{ cdt('xorg-x11-server-xvfb') }}  # [linux]
+        - {{ cdt('libxau') }}                # [linux]
+        - {{ cdt('libxi-devel') }}           # [linux]
+        - occt >=7.4
+        - tbb
+        - freetype
+        - tbb-devel
+        - pybind11
+    host:
+        - python
+        - boost-cpp
+        - ca-certificates
+        - openssl
+        - occt >=7.4
+        - xorg-libxt  # [linux]
+        - xorg-libxpm   # [linux]
+        - xorg-libxaw   # [linux]
+        - xorg-libxfixes  # [linux]
+        - vtk
+    run:
+        - python
+        - occt >=7.4
```

Comment:
I guess `occt` is being used as a shared library? If so it should use pin_compatible to make sure an ABI compatible version is used.

Reply:
yes,  occt is build by one FreeCAD developer.  it should be something like numpy version,  must be matched. 
shall I use  occt == 7.4?

Reply:
I propabaly needs to follow the update of occt version, somehow,  I know occt 7.4 is the only version on conda-forge